### PR TITLE
hook up coveralls

### PR DESCRIPTION
**Pull request by jgravois**
this PR gets headless chrome tests running on travis and should hook up a coveralls integration.

i had trouble with the relevant encrypted environment variables when i tested against [my fork](https://github.com/jgravois/arcgis-rest-js/pull/1), but i figured i'd take one more crack at it. 

ref: https://coveralls.io/github/Esri/arcgis-rest-js

### CODE REVIEW

File: .coveralls.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+git service_name: travis-ci
+repo_token: avn7rbDP1caleMbxtVvI84YrAQxlQaBsp
```
Comment (jgravois) [inline .coveralls.yml]: to be perfectly honest, based on my reading of the coveralls doc, this configuration file [shouldn't be required at all](https://docs.coveralls.io/supported-ci-services). 

that said, i'd be satisfied to let the mystery lie if i could just get a successfull build with:
```yaml
repo_token: $COVERALLS_TOKEN` 
```

Comment (patrickarlt) [reply .coveralls.yml]: I ended up not setting this up like this. I used https://www.npmjs.com/package/coveralls for acetate and just dumped the coverage output to it at the end of the travis CI run like this https://github.com/patrickarlt/acetate/blob/master/.travis.yml#L11

The standard `npm run test:chrome` should already emit a `lcovonly` file in the coverage folder you can pipe to `coveralls`.

Comment (patrickarlt) [reply .coveralls.yml]: @jgravois I think `karma-coveralls` wants it to be `COVERALLS_REPO_TOKEN` see their usage section https://github.com/caitp/karma-coveralls#usage.

Comment (jgravois) [reply .coveralls.yml]: > I think karma-coveralls wants it to be COVERALLS_REPO_TOKEN 

right you are. i've recycled the original token and removed the .yaml configuration file entirely.

**Pull request merged**